diff --git a/src/schema.rs b/src/schema.rs
index f9bfde6..3065702 100644
--- a/src/schema.rs
+++ b/src/schema.rs
@@ -1,10 +1,10 @@
 table! {
     quoths (id) {
-        id -> Integer,
-        author -> Nullable<Integer>,
-        guild -> Nullable<Integer>,
-        starred_by -> Nullable<Integer>,
-        message_id -> Nullable<Integer>,
+        id -> BigInt,
+        author -> Nullable<BigInt>,
+        guild -> Nullable<BigInt>,
+        starred_by -> Nullable<BigInt>,
+        message_id -> Nullable<BigInt>,
         content -> Text,
         legacy -> Bool,
         legacy_author_fallback -> Nullable<Text>,
     }
 }
